src/module/iotjs_module_adc.c

```diff
@@ -234,10 +234,10 @@ JHANDLER_FUNCTION(Unexport) {
 iotjs_jval_t InitAdc() {
   iotjs_jval_t jadc = iotjs_jval_create_object();
 
-  iotjs_jval_set_method(&jadc, "export", Export);
```

Comment:
If you don't have any other special reason to name it, I prefer to leave as it is.